syntax = "proto3";
package m2g.pb;

enum SpriteType {
  NO_SPRITE = 0;
  DWARF_FULL = 1;
  DUNGEON_BEDROCK_1 = 2;
  DUNGEON_BEDROCK_2 = 3;
  DUNGEON_COAL_1 = 4;
  DUNGEON_COAL_2 = 5;
  GRASSLAND_DIRT_WITH_GRASS_1 = 6;
  GRASSLAND_DIRT_WITH_GRASS_2 = 7;
  GRASSLAND_DIRT_1 = 8;
  GRASSLAND_DIRT_2 = 9;
}
